decidim-participatory_processes/app/commands/decidim/participatory_processes/admin/create_participatory_process_group.rb

```diff
@@ -5,13 +5,10 @@ module ParticipatoryProcesses
     module Admin
       # A command with all the business logic when creating a new participatory
       # process group in the system.
-      class CreateParticipatoryProcessGroup < Decidim::Command
-        # Public: Initializes the command.
-        #
-        # form - A form object with the params.
-        def initialize(form)
-          @form = form
-        end
+      class CreateParticipatoryProcessGroup < Decidim::Commands::CreateResource
+        fetch_form_attributes :organization, :title, :description, :hashtag, :group_url, :hero_image,
+                              :developer_group, :local_area, :meta_scope, :participatory_scope, :participatory_structure,
+                              :target, :promoted, :participatory_process_ids
```

Comment:
Is the :participatory_process_ids necessary if we're assigning them in line 42? Seems like we're doing the same thing two times and could save doing one of them

Reply:
removed in 801069f31d